armi/reactor/blocks.py

```diff
@@ -1785,6 +1785,13 @@ def coords(self):
             round(y, units.FLOAT_DIMENSION_DECIMALS),
         )
 
+    def cornersUp(self):
+        """Determine if the hex shape of is corners up or flats up, in relation to the Y axis."""
+        if self.spatialGrid is None:
+            return None
+
+        return self.spatialGrid._unitSteps[0][1] != 0.0
```

Comment:
What about passing off to `HexGrid.cornersUp`?

https://github.com/terrapower/armi/blob/9dbe15c621ecac1060bdfde9715cf87697c24208/armi/reactor/grids/hexagonal.py#L107-L115

```suggestion
        return self.spatialGrid.cornersUp
```